package.json

```diff
@@ -209,6 +213,7 @@
     "uglifyjs": "2.4.11",
     "uglifyjs-webpack-plugin": "2.1.2",
     "wc-sass-render": "1.2.1",
+    "web-component-analyzer": "0.1.20",
```

Comment:
What does `web-component-analyzer` do?

Reply:
It builds on top of jsdoc/tsdoc with a proposed Metadata format for custom elements. https://github.com/w3c/webcomponents/issues/776  This generates markdown docs and API JSON which you can see in this PR. Also it generates a `custom-elements.json` to distribute with our lib which editors like VSCode can understand to give intellisense in HTML templates.